api/src/utils/auth-utils.ts

```diff
@@ -0,0 +1,49 @@
+import { rolesEnum } from '../utils/enums';
+import { SessionUser } from '../utils/helpers';
+
+export const allRoles = [
```

Comment:
Related to the prev comment - maybe consider just using the `rolesEnum` variable here instead of contstructing another array to avoid the hard-coded slicing index